captum/attr/_core/feature_ablation.py

```diff
@@ -286,22 +286,20 @@ def attribute(
             if show_progress:
                 attr_progress.update()
 
+            # number of elements in the output of forward_func
+            n_outputs = initial_eval.numel() if isinstance(initial_eval, Tensor) else 1
+
+            # flattent eval outputs into 1D (n_outputs)
+            # add the leading dim for n_feature_perturbed
+            if isinstance(initial_eval, Tensor):
```

Comment:
is it common for the `forward` not return a Tensor? What else do we expect? I would guess `float` or `double`, but they are all very easy to be converted into tensor and users can just do it by wrapping the forward. So we can explicitly define the return type of `forward`. 

Even if we really want to support non-Tensor returns, can we convert them in `_run_forward` to standardize the logic?